### PR TITLE
Version updates for CI

### PR DESCRIPTION
**Pull request by LorbusChris**

### CODE REVIEW

File: Dockerfile.ci [hunk 1]
```diff
@@ -3,5 +3,7 @@ FROM quay.io/coreos-assembler/fcos:testing-devel
 COPY . /go/src/github.com/openshift/okd-machine-os
 WORKDIR /go/src/github.com/openshift/okd-machine-os
 RUN source /etc/os-release \
-    && sed "s;FEDORA_COREOS_VERSION;${OSTREE_VERSION%.*};g" Dockerfile.template > /tmp/Dockerfile \
+    && sed "s;\${FEDORA_COREOS_VERSION};${OSTREE_VERSION%.*}+okd;g" Dockerfile.template > /tmp/Dockerfile.generated \
```
Comment (vrutkovs) [inline Dockerfile.ci:6]: Not sure if we want both of these changes - I'd leave FCOS version as is and request adding more display names in `oc` to save kernel / crio versions

Comment (LorbusChris) [reply Dockerfile.ci:6]: Hm I think we want to differentiate the plain FCOS version from our OKD build. maybe `+okd.$(date +"%Y%m%d%H%M")` would be even better?

Comment (LorbusChris) [reply Dockerfile.ci]: Removed the commit for now

**Pull request merged**